plugins/vpp/l3plugin/vppcalls/l3_vppcalls.go

```diff
@@ -232,6 +236,15 @@ type TeibVppRead interface {
 	DumpTeib() ([]*l3.TeibEntry, error)
 }
 
+// VrrpVppAPI provides methods for managing VPP VRRP.
+type VrrpVppAPI interface {
+	VppAddVrrp(entry *l3.VRRPEntry) error
+	VppDelVrrp(entry *l3.VRRPEntry) error
+	VppStartVrrp(entry *l3.VRRPEntry) error
+	VppStopVrrp(entry *l3.VRRPEntry) error
+	DumpVrrpEntries() ([]*l3.VRRPEntry, error)
```

Comment:
It would be better for `DumpVrrpEntries` to return `VrrpDetails` type that would have field `VRRPEntry *l3.VRRPEntry` and `Meta *VRRPEntryMeta`, so any extra data from [`vrrp.VrrpVrDetails`](https://github.com/NotSoFancyName/vpp-agent/blob/vrrp-support/plugins/vpp/binapi/vpp2009/vrrp/vrrp.ba.go#L271) message can be stored in there. For example it might be useful to store data from [`vrrp.VrrpVrRuntime`](https://github.com/NotSoFancyName/vpp-agent/blob/vrrp-support/plugins/vpp/binapi/vpp2009/vrrp/vrrp.ba.go#L136-L142).

Reply:
Added rest /dump/vpp/v2/vrrps endpoint for vrrp dump